### PR TITLE
Fix substring operation

### PR DESCRIPTION
**Pull request by dashapaster**

### CODE REVIEW

**Comment by dashapaster** [comment]
@mihaitusa review pls

**Comment by mihaitusa** [comment]
@tethryus Please review this PR...

File: content/io/cloudslang/base/strings/substring.sl [hunk 1]
```diff
@@ -27,12 +27,18 @@
      script: |
         error_message = ""
         max_index=len(origin_string)
-        if end_index==0:
+        if max_index <= 1:
```
Comment (tethryus) [inline content/io/cloudslang/base/strings/substring.sl]: @dashapaster If you set either begin_index or end_index as anything other than integer values ('a' or other illegal characters), the error_message should end with error_message "Invalid values". At the moment, invalid values are handled by the "elif begin_index > max_index-1 or end_index > max_index" condition.

File: test/io/cloudslang/base/strings/substring.inputs.yaml [hunk 2]
```diff
@@ -46,3 +46,25 @@ testSubstringDefault:
...
+
+testSubstringInvalidNegativeInputs:
+  inputs:
+    - origin_string: 'mmmSPAMmmm'
+    - begin_index: -100
+    - end_index: -5
+  description: Tests that substring.sl operation will fail and will finish with a meaningful message when a negative value is given.
+  testFlowPath: io.cloudslang.base.strings.substring
+  outputs:
+    - error_message: "Indexes must be integers"
```
Comment (tethryus) [inline test/io/cloudslang/base/strings/substring.inputs.yaml]: @dashapaster Add the same error_message that the operation outputs when negative indexes are given. Everything else looks ok.

**Comment by mihaitusa** [comment]
@dashapaster  Daria please correct the testSubstringUsingInvalidIndex ... is failing because of the missmatch between error_message expected value and actual value

**Pull request merged**